Deploy/RetrieveWebChatLink.ps1

```diff
@@ -37,24 +37,13 @@ Write-Host "## 1. Retrieve Bot Data from Terraform infrastructure execution"
 $Bot = Get-TerraformOutput("bot") | ConvertFrom-Json
 $success = $success -and $?
 
-# 2. Retrive DirectlineKey and register it to KV
-Write-Host "## 2. Retrive DirectlineKey and register it to KV"
-$directline = $(az bot directline show --resource-group $Bot.resource_group --name $Bot.name --with-secrets true) | ConvertFrom-Json
-$success = $success -and $?
-# create a secret in Key Vault called DirectlineKey
-az keyvault secret set --vault-name $Bot.name --name 'DirectlineKey' --value $directline.properties.properties.sites.key > $null
-
-# 3. Display Bot Name, Endpoint and generate Link
-Write-Host "## 3. Display Bot Name, Endpoint and generate Link"
-$endpoint = "https://$($Bot.name).trafficmanager.net"
-$success = $success -and $?
-$queryparams = "?bot=$($Bot.name)&endpoint=$($endpoint)"
-$webchathtmlfile = Get-ItemProperty -Path "$(Get-ScriptPath)/../WebChat/index.html"
+# 2. Generate Link
```

Comment:
The endpoint will be different depending if the user chooses to use a custom domain. How about retrieving the message endpoint URL via `az bot` and removing `/api/messages` from it?

Reply:
Oh I see! It is a good point! I will update this part! 😉